steps/server12.js

```diff
@@ -0,0 +1,37 @@
+// Now we have to create the methods for the front to communicate with the backend
```

Comment:
Learners should change something each step. As I wrote in another comment, in the final version we should have learners write as much of the code as possible, even on the front end. I imagine that when learners get to this point they could start working on `/public/script.js` to get things working between the front and backend.